definitions/json/EventToComplexTypes/EventToComplexTypes.json

```diff
@@ -6369,7 +6369,17 @@
     "CaseFieldID": "referralCollection",
     "ListElementCode": "isUrgent",
     "EventElementLabel": " ",
-    "FieldDisplayOrder": 2,
+    "FieldDisplayOrder": 3,
+    "DisplayContext": "OPTIONAL",
+    "Publish": "Y"
+  },
+  {
+    "ID": "ReferralDetails",
```

Comment:
If you are going to have a referral number, does it make sense to have it as an optional field? The user could not enter a number for it. Does it make sense to hide it and autopopulate it via callbacks? Unless I've missed something?

Reply:
Referral Number is already filled out by callbacks, the important part for WA is the Publish so that the value is passed to the DMN files.

Reply:
Referral Number is already filled out by callbacks, the important part for WA is the Publish so that the value is passed to the DMN files.